src/auto-complete.jsx

```diff
@@ -298,8 +298,8 @@ const AutoComplete = React.createClass({
   },
 
   handleKeyDown(event) {
-    switch (event.keyCode) {
-      case KeyCode.ENTER:
+    switch (keycode(event.keyCode)) {
```

Comment:
Why not using `keycode(event)`?
(https://github.com/timoxley/keycode/blob/master/index.js#L17)